### PR TITLE
LPS-156961 Extract Filter Modal from Object View

### PR DESCRIPTION
**Pull request by danvb112**

### CODE REVIEW

**Comment by danvb112** [comment]
ci:test:relevant

**Review by naraandrade — CHANGES_REQUESTED** [review]

**Comment by danvb112** [comment]
ci:test:sf

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/FilterScreen/FilterScreen.tsx [hunk 1]
```diff
@@ -63,14 +93,27 @@ export function FilterScreen() {
...
 			)}
 		</>
 	);
 }
+
+interface IItem extends TLabelValueObject {
+	checked?: boolean;
+}
+
+type TLabelValueObject = {
```
[the remark this answers is not in the document]
Comment (danvb112) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectView/FilterScreen/FilterScreen.tsx]: Hey @naraandrade, I added a new `LabelValueObject` interface in `index.d.ts` and changed it everywhere that was used as a `type` before

**Comment by danvb112** [comment]
ci:test:sf

**Comment by danvb112** [comment]
ci:forward

**Pull request closed without merging**